src/strategies/index.ts

```diff
@@ -407,6 +409,8 @@ const strategies = {
   'digitalax-mona-quickswap': digitalaxMonaQuickswap,
   'digitalax-genesis-contribution': digitalaxGenesisContribution,
   'digitalax-lp-stakers': digitalaxLPStakers,
+  'digitalax-lp-stakers-matic': digitalaxLPStakersMatic,
+  'digitalax-mona-stakers-matic': digitalaxMonaStakersMatic,
```

Comment:
Would be great if you can do separate PRs for two strategies. will be easy to test/review/integrate